addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6451,22 +6459,22 @@ msgstr ""
 
 #: system/settings/darwin.xml
 msgctxt "#13612"
-msgid "Universal Remote"
+msgid "Universal remote"
 msgstr ""
 
 #: system/settings/darwin.xml
 msgctxt "#13613"
-msgid "Multi Remote (Harmony)"
+msgid "Harmony remote"
 msgstr ""
 
 #empty strings from id 13614 to 13619
 
 msgctxt "#13620"
-msgid "Apple Remote Error"
+msgid "Apple remote error"
```

Comment:
You are wrong. "Apple Remote" is the name of the specific product those labels are referring to. Has nothing to do with any other occurences of the word "remote".